tutorial/README.md

```diff
@@ -486,6 +487,27 @@ INSERT INTO informix.customers(first_name, last_name, email) VALUES ('John', 'Do
 docker-compose -f docker-compose-ifx.yaml down
 ```
 
+## Using MariaDB
+
+```shell
+# Start the topology as defined in https://debezium.io/documentation/reference/stable/tutorial.html
+export DEBEZIUM_VERSION=3.1
+docker-compose -f docker-compose-mariadb.yaml up
+
+# Start MariaDB connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @register-mariadb.json
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-mariadb.yaml exec kafka /kafka/bin/kafka-console-consumer.sh \
+    --bootstrap-server kafka:9092 \
+    --from-beginning \
+    --property print.key=true \
+    --topic dbserver1.inventory.customers
+
```

Comment:
@kmos COuld you please add an instruction how to modify a database like in https://github.com/debezium/debezium-examples/blob/main/tutorial/README.md#using-mysql ?

Reply:
yes!